lambdas/restore-vcs/src/main/java/uk/gov/di/ipv/core/restorevcs/RestoreVcsHandler.java

```diff
@@ -44,17 +48,20 @@ public class RestoreVcsHandler implements RequestStreamHandler {
     private static final Logger LOGGER = LogManager.getLogger();
     private final ConfigService configService;
     private final VerifiableCredentialService verifiableCredentialService;
+    private final DataStore<VcStoreItem> vcDataStore;
```

Comment:
If you're doing it 'directly' with a data store, do you still need the `VerifiableCredentialService`?